www/doc/1.7/report.html

```diff
@@ -20,143 +22,186 @@ <h1>Herbie reports</h1>
     </nav>
   </header>
 
-  
   <h1>The Herbie report</h1>
 
   <p>When used <a href="using-web.html">in the browser</a>, Herbie
-  generates HTML reports full of information about the accuracy of
-  your input and its output expressions.</p>
+    generates HTML reports full of information about the accuracy and relative 
+    speed of your input and its output expressions.</p>
+
+  <h2 id="links">Additional links</h2>
+
+  <p>The top of the report page has a right-hand menu bar with additional
+    links. “Report” returns you to Herbie's main page.
```

Comment:
@pavpanchekha  Aren't we also planning on removing "Report" from this version? 

NIT: On line 37 the graph.html page is referred to as the report. Which to me a is a little confusing considering there is also a report.html page generated seems to be more of a summary of multiple reports. And timeline.html is referred to by the Metrics link.

Reply:
I think we're planning to not show the report link if you run the web version without logging, but the user might run reports or the web version with logging, in which case that link will be there. So we should still describe it (though it would be good to note that it's only present in some modes).

I agree that the names are confusing. I guess we've been calling the pages the "report" and "details" page? Though I agree that it's confusing.

Reply:
Well got tripped up my caching for a bit but made changes so that the "Report" menu shows when `--save-session` is passed in. Let me know if that's what you were thinking?